HeliPort/NetworkManager+Data.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  NetworkManager+Data.swift
+//  HeliPort
+//
+//  Created by 梁怀宇 on 2020/3/23.
+//  Copyright © 2020 lhy. All rights reserved.
+//
+
+/*
+ * This program and the accompanying materials are licensed and made available
+ * under the terms and conditions of the The 3-Clause BSD License
+ * which accompanies this distribution. The full text of the license may be found at
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import Foundation
+
+final class NetworkInfo {
+    let ssid: String
+    let isConnected: Bool
+    var rssi: Int
+
+    var auth = NetworkAuth()
+
+    enum AuthSecurity: UInt32 {
+        case NONE        = 0x00000000
+        case USEGROUP    = 0x00000001
```

Comment:
We don't need the `AuthSecurity` enum anymore. Btw, can you put the description extension of `itl80211_security` enum together with that of `itl_phy_mode` enum?